docs/configuration/settings.rst

```diff
@@ -128,7 +128,7 @@ Storage
 
 .. code-block:: ini
 
-    kinto.storage_backend = kinto.core.storage.redis
+    kinto.storage_backend = kinto_redis.storage
```

Comment:
*me thinking out loud*: Since those settings values are given as examples, I think we could give a fully qualified postgres config instead?